__snapshot__/correct/blocks_hard.cac.resolved.snap

```diff
@@ -1,3 +1,14 @@
 {
-  "__class__": "EmptyMap"
+  "__class__": "SingletonMap",
+  "(Location(value\u003d90), Location(value\u003d90))": {
```

Comment:
These "value\u003d90" look quite strange, is it intended?

Reply:
This value was displayed the same way on previous snapshots, so I thought it is OK